src/ray/protobuf/gcs.proto

```diff
@@ -703,6 +703,7 @@ message JobTableData {
   // The optional JobInfo from the Ray Job API.
   optional JobsAPIInfo job_info = 10;
   // Whether this job has running tasks.
+  // In GetAllJobInfo, if GCS can't reach the driver, it will set this field to true.
```

Comment:
Actually after a second thought, can we do

```
optional bool is_running_tasks
```

and don't set it for timeout. I think this is more explicit.

Reply:
this will be a breaking change (required -> optional). but ok since nobody other than humans are reading it. will do